image/templates/helm/stackrox-central/values-public.yaml.example.htpl

```diff
@@ -451,6 +451,240 @@
+#    # or lower if autoscaling is enabled (see below).
+#    replicas: 2
+#
+#    # Settings related to autoscaling the matcher deployment.
+#    autoscaling:
+#      # If true, autoscaling will be disabled. None of the other settings in this section will
+#      # have any effect.
+#      disable: false
+#      # The minimum number of replicas for autoscaling. The following value is the default.
+#      minReplicas: 2
+#      # The maximum number of replicas for autoscaling. The following value is the default.
+#      maxReplicas: 3
+#
+#    # Resource settings for the matcher deployment.
+#    resources:
+#      requests:
+#        memory: "4Gi"
+#        cpu: "1000m"
+#      limits:
+#        memory: "5Gi"
+#        cpu: "2000m"
+#
+#  # Configuration for the DB component:
+#  db:
+#
+#    # If you want to enforce indexer to only run on certain nodes, you can specify
+#    # a node selector here to make sure Scanner V4 can only be scheduled on Nodes with the
+#    # given label.
+#    nodeSelector:
+#      # This can contain arbitrary `label-key: label-value` pairs.
+#      role: scanner-v4
+#
+#    # If scheduling needs specific affinities, you can specify the corresponding affinities here.
+#    affinity:
+#      nodeAffinity:
+#        preferredDuringSchedulingIgnoredDuringExecution:
+#        # Scanner V4 DB is single-homed, so avoid preemptible nodes.
+#        - weight: 100
+#          preference:
+#            matchExpressions:
+#            - key: cloud.google.com/gke-preemptible
+#              operator: NotIn
+#              values:
+#                - "true"
+#        - weight: 50
+#          preference:
+#            matchExpressions:
+#            - key: node-role.kubernetes.io/infra
+#              operator: Exists
+#        - weight: 25
+#          preference:
+#            matchExpressions:
+#            - key: node-role.kubernetes.io/compute
+#              operator: Exists
+#        # From v1.20 node-role.kubernetes.io/control-plane replaces node-role.kubernetes.io/master (removed in
+#        # v1.25). We apply both because our goal is not to run pods on control plane nodes for any version of k8s.
+#        - weight: 100
+#          preference:
+#            matchExpressions:
+#            - key: node-role.kubernetes.io/master
+#              operator: DoesNotExist
+#        - weight: 100
+#          preference:
+#            matchExpressions:
+#            - key: node-role.kubernetes.io/control-plane
+#              operator: DoesNotExist
+#
+#    # Resource settings for the DB deployment.
+#    resources:
+#      requests:
+#        cpu: "200m"
+#        memory: "2Gi"
+#      limits:
+#        cpu: "2000m"
+#        memory: "4Gi"
+#
+#    # Configure DB password. If `generate` is true, a new password will be
+#    # generated. If it is false, a valid password is expected in `value`.
+#    password:
+#      generate: null
+#      value: null
+#
+#    # Configure persistence backend for the DB.
```

Comment:
Maybe it's worth elaborating on the options like you did in secured cluster example values file.

Reply:
Actuall I have skipped this on purpose, because a user really should be using a PVC, especially on central side.

I see the `values-public.yaml.example.htpl` file not as a complete reference to all possible values (this is in `values.yaml.htpl` for central), but more as a collection of settings guiding the user into the right direction, highlighting the correct way of doing things.

Reply:
btw, I also found it a bit tricky to understand the subtle differences between all these values files, but as I understand it, this is it.

Reply:
Sounds reasonable.